apps/pushes/views/diff.py

```diff
@@ -172,104 +173,57 @@ def diffLines(self, path, action):
             p = getParser(path)
         except UserWarning:
             return None
-        if action == 'added':
-            a_entities = []
-            a_map = {}
-        else:
+        old_data = OrderedDict()
```

Comment:
What kind of `data` is this? `diff_data`, `flat_data`, `ordered_entities`?

Reply:
I think the difference between old and new is the relevant qualifier here. It maps from key-ish thing to value. To answer the question you didn't ask, if you need the variable name to be more specific, this needs a comment, not a variable name. I'll add a comment.

Reply:
I added a comment to the top of the file to describe the algorithm.

We also talked about where to put the OrderedDict, and I went for the location where it was, but the .parse function now yields keys and values, so I can still call .update() on the result.

Reply:
> if you need the variable name to be more specific, this needs a comment, not a variable name. I'll add a comment.

This is a bad practice. Virtually every other name is better than `data`. How about `old_entities` and `new_entities`?

Reply:
Well, the challenge is that this isn't something that shows up anywhere else. Entities would be wrong, as these are just values, and not even Entity.vals (due to FluentAttribute not being an Entity). `old_translations` vs `new_translations` could work?

Reply:
Yeah, that's better too, thanks. Or perhaps `diffables`?